packages/twenty-server/src/core/messaging/timeline-messaging.service.ts

```diff
@@ -14,7 +24,11 @@ export class TimelineMessagingService {
   async getMessagesFromPersonIds(
     workspaceId: string,
     personIds: string[],
+    page: number = 1,
```

Comment:
Can you create a messaging.constants.ts. and store default values such as page size etc, it will be easier to change that later if needed if it's referenced in multiple places